lib/subscription/subscription-store.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference path='../../typings/tsd.d.ts' />
+
+import _ = require('lodash');
+import Subscription = require('./subscription');
+
+export = SubscriptionStore;
+
+class SubscriptionStore<T extends Subscription> {
```

Comment:
If we do decide to keep this, I would make this a normal class and remove the generic since it **only** operations on `Subscription`.